spec/features/repositories/repository_settings_spec.rb

```diff
@@ -62,8 +62,11 @@
       find('a.icon-delete', text: I18n.t(:button_delete)).click
 
       # Confirm the notification warning
-      warning = (type == 'managed') ? '-warning.-severe' : '-warning'
-      expect(page).to have_selector(".notification-box.#{warning}")
+      if type == 'managed'
+        expect(page).to have_selector("form.-danger-zone")
+      else
+        expect(page).to have_selector(".notification-box.-warning")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.